assets/queries/terraform/azure/aks_uses_disk_encryption_set/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "b17d8bb8-4c08-4785-867e-cb9e62a622aa",
+  "queryName": "AKS Uses Disk Encryption Set",
```

Comment:
The query name could be more intuitive like "AKS Disk Encryption Set ID Undefined" (just a suggestion)

Reply:
I agree. Don't forget to change the directory name too.